python/rmm/_cuda/11.0/gpu.pxi

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
-# For CUDA 11.0
+# For CUDA 11.0, 11.1 and 11.2
```

Comment:
Should we rename this folder to capture this?

Reply:
Changed to `11.x`